projects/core/src/occ/adapters/customer-coupon/customer-coupon-occ.module.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
```

Comment:
No need for this file I think, because the implementation of CustomerCouponAdapter is defined in user-occ.module.ts

Reply:
updated